packages/near-fast-auth-signer-e2e-tests/tests/auth.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { KeyPair } from 'near-api-js';
+
+import PageManager from '../pages/PageManager';
+import { test } from '../test-options';
+import { getRandomEmailAndAccountId } from '../utils/email';
+import { rerouteToCustomURL } from '../utils/url';
+
+test.beforeEach(async ({ page, baseURL, relayerURL }) => {
+  await page.goto(baseURL);
+
+  // Overwrite relayer URLs to use local instance
```

Comment:
Why not use the `relayerUrl` param in `setupFastAuthWallet` or use [the `setRelayerUrl` method](https://github.com/near/near-fastauth-wallet/blob/main/src/lib/near-fastauth-wallet.ts#L346) here?

Reply:
My idea was to take advantage of the playwright projects: https://playwright.dev/docs/test-projects#configure-projects-for-multiple-environments

So we configure local, testnet, mainnet, etc... projects directly from `playwright.config.ts` without changing the App code. 

I think it's easier and more clear. What do you think?